cpp/cppsdk/include/playfab/PlayFabBaseModel.h

```diff
@@ -78,7 +78,7 @@ namespace PlayFab
     // Utilities for [de]serializing time_t to/from json
     inline void ToJsonUtilT(const time_t input, Json::Value& output)
     {
-        input; // vs is stupid
+        (void)input; // vs is stupid
```

Comment:
nit: since it's a bit meta about the compiler tools, Can we remove the // vs is stupid?